src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -6,23 +6,31 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.File.ReadImage</oldName>
-    <newName>DSCore.IO.File.ReadImage</newName>
+    <newName>DSCore.IO.FileSystem.ReadImage</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.IO.File.ReadImage</oldName>
+    <newName>DSCore.IO.FileSystem.ReadImage</newName>
   </priorNameHint>
```

Comment:
Both of these migrations are required because the older migration places an obsolete node which is actually part of a test.  The newest version also has a didn't amount of input ports which causes additional failures if used instead.  I am guessing this is why the obsolete node was never removed.

Reply:
same goes for `WriteImage` below

Reply:
so what did you decide to do with the obsolete nodes? old nodes are just migrated to the obsolete ones with an updated message?

Reply:
The old nodes migrate to obsolete nodes because the newest version of the nodes has different inputs.  The node appears with a warning message stating you should use the `Image` nodes instead